projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/cancellation-return-items/cancellation-return-items.component.html

```diff
@@ -110,11 +108,19 @@
                   }}
                 </div>
                 <div [formGroupName]="i">
+                  <cx-item-counter
+                    *ngIf="!confirmRequest"
+                    [isValueChangeable]="true"
+                    [step]="1"
+                    [min]="0"
+                    [max]="item.returnableQuantity"
+                    (update)="updateQty()"
+                    formControlName="quantity"
+                  >
+                  </cx-item-counter>
                   <input
-                    type="text"
+                    *ngIf="confirmRequest"
```

Comment:
Maybe this could be an item counter with isValueChangeable false. It will have the same behavior but a more consistent UI.

Reply:
yes!